app/config.js

```diff
@@ -23,7 +23,7 @@ window.OPENSHIFT_CONFIG = {
   },
   additionalServers: [{
     protocol: "https",
-    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io:6443",
+    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io",
```

Comment:
we can comment it out. I was hoping to be able to ignore the errors by adding a flag to an additional server block but that PR (https://github.com/openshift/origin-web-common/pull/22) is still in limbo.